src/components/sections/SchoolsPage/Pagination.tsx

```diff
@@ -39,37 +39,39 @@ const PaginationSkipButton = styled(BaseButton)`
   }
 `;
 
-const Pagination: FC<any> = ({ count, page, onPageChange }) => {
+const Pagination: FC<any> = ({ count, page, onPageChange, disabled }) => {
   const totalPages = getTotalPages(count);
 
+  if (totalPages === 1) return null;
```

Comment:
So when there are no results it shows up, disabled, and if there's only one page it does now show up at all?

Reply:
* When there is one page is does not show up at all
* When `data` variable is falsy the request is pending. (It's how SWR handles it. Like in this example -> https://swr.vercel.app/). Changed from `!schools` to `!data`. My bad.